ideascube/library/migrations/0008_library_stock.py

```diff
@@ -26,6 +30,13 @@ def migrate_data(apps, schema_editor):
         new_book.save()
 
         for old_bookspecimen in old_book.specimens.all():
+            if old_bookspecimen.serial in specimens:
+                # If "old serial" equals "new barcode" then we are dealing with
+                # the same specimen
+                old_specimen = specimens.pop(old_bookspecimen.serial)
+                old_specimen.item.delete()
+                old_specimen.delete()
```

Comment:
Should we delete the old_specimen or update it ?
I see at least InventorySpecimen that has a ForeignKey on the Specimen. What happen if we delete a specimen referenced by a InventorySpecimen ?